apps/account-api/src/controllers/v1/accounts-v1.controller.ts

```diff
@@ -1,12 +1,13 @@
-import { AccountsService } from '#api/services/accounts.service';
-import { AccountResponseDto } from '#lib/types/dtos/accounts.response.dto';
-import { WalletLoginRequestDto } from '#lib/types/dtos/wallet.login.request.dto';
-import { WalletLoginConfigResponseDto } from '#lib/types/dtos/wallet.login.config.response.dto';
-import { WalletLoginResponseDto } from '#lib/types/dtos/wallet.login.response.dto';
+import { AccountsService } from '#account-api/services/accounts.service';
+import { AccountResponseDto } from '#account-lib/types/dtos/accounts.response.dto';
+import { WalletLoginRequestDto } from '#account-lib/types/dtos/wallet.login.request.dto';
+import { WalletLoginConfigResponseDto } from '#account-lib/types/dtos/wallet.login.config.response.dto';
+import { WalletLoginResponseDto } from '#account-lib/types/dtos/wallet.login.response.dto';
 import { Body, Controller, Get, Post, HttpCode, HttpStatus, Logger, Param, HttpException } from '@nestjs/common';
 import { ApiBody, ApiOkResponse, ApiCreatedResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
-import { ConfigService } from '#lib/config';
+import { ConfigService } from '#account-lib/config';
 
+const foo = 'bar';
```

Comment:
Junk line to remove. Might have been for triggering CI.
```suggestion
```

Reply:
Was testing linter.